apps/mobile/src/packages/quiz/quiz-api.ts

```diff
@@ -0,0 +1,34 @@
+import { APIPath, ContentType } from "~/libs/enums/enums";
+import { BaseHttpApi } from "~/libs/packages/api/api";
+import { type HTTP } from "~/libs/packages/http/http";
+import { type Storage } from "~/libs/packages/storage/storage";
+
+import { QuizApiPath } from "./libs/enums/enums";
+import { type QuizQuestionDto } from "./libs/types/types";
+
+type Constructor = {
+	baseUrl: string;
+	http: HTTP;
+	storage: Storage;
+};
+
```

Comment:
Let's move this type to libs/types/api or even to shared as suggested @katerynakostikova